### ISSUE TITLE
[Programatic Access - Make Code - Home page] 'Name and Role' is not define for close button in 'Details dialog box'.

### ISSUE THREAD

**Issue by AtulAgrahari** [microsoft/pxt-microbit#2349]
**User Experience:**
When standard controls from accessible technologies are not used and if the user interface elements are not used according to specification, Assistive Technologies (AT) will not be able to gather information about, activate(or set) and keep up to date on the status of user interface controls in the content.

**Repro Steps:**
1. Launch application URL : https://makecode.microbit.org/beta
2. Home page will open.
3. Navigate to the controls appearing under "Tutorials, Games and Radio Games" section in Home page.
4. Select any option and navigate to the details dialog box.
5. Verify the issue.

**Actual Result:**
 'Name and Role' is not define for close button in 'Details dialog box'.
Nothing narrated by narrator and JAWS and NVDA narrated like: Blank, when focus is on 'Close' button.

**Expected Result:**
 'Name and Role' should be define for close button in 'Details dialog box'.

**Note:**
1. Same issue is also exists with Chrome.
2. This issue is exists for all ‘Close’ button present under all dialog box in home page.

**MAS Reference:**
MAS 4.1.2 - Name, Role, Value (40B)

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

![MAS4 1 2_Close](https://user-images.githubusercontent.com/53811418/62535368-dae41100-b868-11e9-9ec7-3431c894a33d.png)

**Comment by abchatra** [comment]
Please re-test using beta site https://makecode.microbit.org/beta . If it reproduces let me know. I will reopen the bug.

**Comment by AtulAgrahari** [comment]
Issue still exists in https://makecode.microbit.org/beta so reactivating the issue.

**Comment by sonimishra** [comment]
Issue has been verified and now name and control type is defied for the control.
Hence closing the bug.
